src/jet/main.clj

```diff
@@ -94,7 +94,9 @@
                 :interactive :collect
                 :edn-reader-opts
                 :help]} (parse-opts args)]
-    (cond version
+    (if (nil? args)
```

Comment:
I think `(nil? args)` could be moved into the `cond`

Reply:
Right, yes! That would work!

Reply:
Changed it!